public/index.html

```diff
@@ -20,6 +20,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Delivery Console</title>
+    <script src="https://cdn.auth0.com/js/auth0/8.9/auth0.min.js"></script>
```

Comment:
Is there a reason to pull this from a CDN over Yarn or NPM?

Reply:
nope, absolutely none :D